package.json

```diff
@@ -3,18 +3,43 @@
   "version": "1.0.0",
   "description": "dxw Playbook",
   "scripts": {
-    "start": "docsify serve .",
-    "lint": "remark .",
+    "markdown:lint": "remark .",
     "vale": "~/go/bin/vale */*.md *.md",
-    "vale-suggestions": "~/go/bin/vale --minAlertLevel suggestion */*.md *.md "
+    "vale-suggestions": "~/go/bin/vale --minAlertLevel suggestion */*.md *.md ",
+    "build:dev": "webpack --mode=development",
+    "build": "webpack -p --mode=production",
+    "watch": "webpack --mode=development --watch",
+    "js:lint": "standard '_javascripts/src/**/.js' '_spec/*.js'",
+    "js:lint:fix": "standard '_javascripts/src/**/.js' '_spec/*.js' --fix",
+    "prettier": "prettier --check '_javascripts/src/**/*.js' '_spec/*.js'",
+    "prettier:fix": "prettier --write '_javascripts/src/**/*.js' '_spec/*.js'",
+    "test": "jest"
   },
   "dependencies": {
-    "docsify": "^4.5.5",
-    "docsify-cli": "^4.1.12",
+    "@babel/core": "^7.10.2",
+    "@babel/preset-env": "^7.10.2",
+    "babel": "^6.23.0",
+    "babel-loader": "^8.1.0",
+    "core-js": "^3.6.5",
+    "jest": "^26.0.1",
     "marked": "^1.0.0",
+    "prettier": "^2.0.5",
     "prismjs": "^1.9.0",
     "remark-cli": "^8.0.0",
     "remark-lint": "^7.0.0",
-    "remark-preset-lint-markdown-style-guide": "^3.0.0"
+    "remark-preset-lint-markdown-style-guide": "^3.0.0",
+    "standard": "^14.3.4",
```

Comment:
Prefer ESLint over standard for JavaScript.

Reply:
👍